src/sbFetch.ts

```diff
@@ -30,8 +30,7 @@ class SbFetch {
 		this.headers = $c.headers || new Headers()
 		this.timeout = $c?.timeout ? $c.timeout * 1000 : 0
 		this.responseInterceptor = $c.responseInterceptor
-		this.fetch = $c.fetch || fetch
-		this.ejectInterceptor = false
+		this.fetch = (...args: Parameters<typeof fetch>) => ($c.fetch ? $c.fetch(...args) : fetch(...args))
```

Comment:
I think this will solve the TypeScript error: 

```suggestion
		this.fetch = $c.fetch ?? fetch
```

![image](https://github.com/storyblok/storyblok-js-client/assets/14206504/8ce8f3b3-4e86-4d29-884c-a3207d13c0e9)